packages/create-cloudflare/templates/analog/templates/src/dev-bindings.ts

```diff
@@ -0,0 +1,18 @@
+import { NitroApp } from 'nitropack';
+import { defineNitroPlugin } from 'nitropack/dist/runtime/plugin';
+
+export default defineNitroPlugin((nitroApp: NitroApp) => {
+  nitroApp.hooks.hook('request', async (event) => {
+    const _pkg = 'wrangler'; // Bypass bundling!
+    const { getPlatformProxy } = (await import(
+      _pkg
+    )) as typeof import('wrangler');
+    const platform = await getPlatformProxy();
+
+    event.context.cf = platform['cf'];
+    event.context.cloudflare = {
+      env: platform['env'] as unknown as Env,
+      context: platform['ctx'],
```

Comment:
in addition to this, we should really expose the bindings as Angular providers as well, similar to how it's done here: https://github.com/analogjs/analog/pull/1020/files#diff-c7583ca9a43860e4a6bb2c9836d9506b1979a5e8822b31dbb19a90efd1e9e351R31

maybe we should really land https://github.com/analogjs/analog/pull/1020 first and then build the c3 template on top of that?

Reply:
https://github.com/analogjs/analog/pull/1020 has landed and it's part of Analog [v.1.2.0](https://github.com/analogjs/analog/releases/tag/v1.2.0) 🙂

Reply:
I've updated to `analog@1.2.0` which landed yesterday evening. If there's any additional gap in the Angular providers this is something that should get handled on the analog side.